### ISSUE TITLE
Cannot use hyperspyUI with latest hyperspy beta release

### ISSUE THREAD

**Issue by jat255** [hyperspy/hyperspyUI#155]
Installing the latest `RELEASE_next_minor` version of Hyperspy breaks HyperspyUI. This happens on Windows and Linux on Python 3.5 and 3.6, as far as I can tell.

### Excpected behavior

Should work as expected

### Actual behavior

The main interface loads okay, but when opening a signal, the plots are plotted externally in their own backend windows (complete with the `matplotlib` control buttons), rather than within the main interface window. The relevant controls of hyperspyUI (such as the element picker) do not become active (see photo).

![image](https://user-images.githubusercontent.com/1278301/37927796-04fab5ae-3109-11e8-99ae-fe1b05683a94.png)

**Comment by vidartf** [comment]
In chat you wrote:

```
C:\anaconda3\envs\hyperspy\lib\site-packages\matplotlib\__init__.py:1401: UserWarning:
   This call to matplotlib.use() has no effect because the backend has already been chosen;
matplotlib.use() must be called *before* pylab, matplotlib.pyplot, or
matplotlib.backends is imported for the first time.

warnings.warn(_use_error_msg)
```

I believe this is significant. To me, it indicates that the way hyperspy (or another dependency) imports matplotlib has changed.

**Comment by jat255** [comment]
See https://github.com/hyperspy/hyperspyUI/pull/150#issuecomment-377564846:

In that issue, this problem is caused by an import of `pyplot` by `mrcz`. Testing now to see if that's the case on in this issue as well.

**Comment by ericpre** [comment]
The last version of mrcz (0.3.6) needs to be install to fix the issue.

**Comment by vidartf** [comment]
Is this resolved by #150?

**Comment by jat255** [comment]
Yes, I think this is resolved; if not by #150, then by updating `mrcz`
